recipe/meta.yaml

```diff
@@ -12,48 +12,47 @@ source:
 
 build:
   number: 1000
-  skip: true  # [win]
+  skip: true  # [win and vc<14]
+  run_exports:
+    # https\://abi-laboratory.pro/\?view\=timeline\&l\=ilmbase
+    - {{ pin_subpackage(name|lower, max_pin='x.x') }}
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake  # [win]
-    - ninja  # [win]
 
   host:
     - zlib
 
-  run:
-    - zlib
-
```

Comment:
zlib has a run_exports section so it is not necessary to repeat it from the host section.